system/Commands/Utilities/Optimize.php

```diff
@@ -99,32 +115,73 @@ private function removeFile(string $cache): void
         }
     }
 
-    private function enableCaching(): void
+    private function enableCaching(?bool $enableConfigCache, ?bool $enableLocatorCache, ?bool $disable): void
```

Comment:
Why the method `enableCaching()` does disable caching?
It is strange and not intuitive.
Please add a new private method to disable caching.

Reply:
Done